core/Updater/Migration/Db/Sql.php

```diff
@@ -41,7 +43,8 @@ public function __construct($sql, $errorCodesToIgnore)
         }
 
         $this->sql = $sql;
-        $this->errorCodesToIgnore = $errorCodesToIgnore;
+        $globalErrorCodesToIgnore = Config::getInstance()->database['ignore_error_codes'];
```

Comment:
This throws a notice/error, as the config options isn't defined by default.
```suggestion
        $globalErrorCodesToIgnore = Config::getInstance()->database['ignore_error_codes'] ?? [];
```